ansible/roles/copy-authorized-keys/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+public_keys: public_keys
```

Comment:
Should this be an empty list, rather than a string?

Reply:
If we are implementing your favourite role Stig speaks of below, I could remove this role altogether.